src/components/users/userProfile/UserProfile.js

```diff
@@ -1,35 +1,38 @@
-import React from 'react';
-import { useState, useEffect } from 'react';
-import client from '../../../utils/client';
-import { useParams } from 'react-router-dom';
-import Header from '../../Header/Header';
-
+import React from "react";
+import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
+import client from "../../../utils/client";
+import { useParams } from "react-router-dom";
+import Header from "../../Header/Header";
+import './style.css';
 
 const UserProfile = () => {
-
-  const [profile, setProfile] = useState('');
+  const [profile, setProfile] = useState("");
   const { id } = useParams();
-  let role = profile.role
+  let role = profile.role;
 
   useEffect(() => {
-      client
-        .get(`/user/${id}`)
-        .then((res) => {
-          setProfile(res.data.data.user);
-        })
-        .catch((err) => console.log(err.response));
+    client
+      .get(`/user/${id}`)
+      .then((res) => {
+        setProfile(res.data.data.user);
+      })
+      .catch((err) => console.log(err.response));
   }, [id]);
 
   return (
     <div>
-      <Header role={role}/>
+      <Header role={role} />
       <h1>User Profile</h1>
       <h2>
         {profile.firstName} {profile.lastName}
       </h2>
       <p>Email: {profile.email}</p>
       <p>Bio: {profile.biography}</p>
-      <p>Github: {profile.githubUrl}</p>
+      <p>Github: {profile.github_url}</p>
```

Comment:
Remember it's possible for a user to view profiles of other users, not just their own - should this link only be displayed for users when viewing their own profile?

Reply:
Yeah, I can see the links for other users github url's in mine. That's right isn't it?

Reply:
Ah sorry I was referring to the Edit link (my bad!) - it should only be possible to edit my own profile.